modules/nw-rfhe-introduction.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * monitoring/using-rfhe.adoc
+
+:_content-type: CONCEPT
+[id="nw-rfhe-introduction_{context}"]
+= How Redfish hardware events work
+
+The Baremetal Hardware Event Proxy enables applications running on baremetal clusters to respond quickly to Redfish hardware changes and failures such as breaches of temperature thresholds, fan failure, disk loss, power outages, and memory failure. These hardware events are delivered over a reliable low-latency transport channel based on Advanced Message Queuing Protocol (AMQP). The latency of the messaging service is between 10 to 20 milliseconds.
```

Comment:
Flagging the first sentence for inconsistent use of "baremetal". You use "bare-metal" later in this text.

Reply:
Agree. It should be two words. We sometimes write "bare-metal cluster" and sometimes "bare metal cluster". IMO, both of the two-word phrasings are fine. ISG suggests hyphenating the two-word adjective if it brings more clarity:
> Use hyphens in compound adjectives to show which words modify another word if it is necessary to avoid ambiguity